vaadin-time-picker-flow/src/test/java/com/vaadin/flow/component/timepicker/tests/TimePickerTest.java

```diff
@@ -15,18 +15,39 @@
  */
 package com.vaadin.flow.component.timepicker.tests;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
 import java.time.Duration;
 import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
 
+import org.junit.Assert;
 import org.junit.Test;
+import org.mockito.Mockito;
 
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.timepicker.GeneratedVaadinTimePicker;
 import com.vaadin.flow.component.timepicker.TimePicker;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
+import com.vaadin.flow.di.Instantiator;
+import com.vaadin.flow.dom.Element;
+import com.vaadin.flow.server.VaadinService;
+import com.vaadin.flow.server.VaadinSession;
 
 public class TimePickerTest {
 
+    private static LocalTime TEST_VALUE = LocalTime.now();
+
+    private static class TestTimePicker
+            extends GeneratedVaadinTimePicker<TestTimePicker, LocalTime> {
+
+        TestTimePicker() {
```

Comment:
Does it make sense to test the case with mandatory init value (`TimePicker(LocalTime time)`) ?

Reply:
There is already such test `setInitialValue`